_docs/mini_memcheck.md

```diff
@@ -170,10 +170,10 @@ Sentinel value is a chunck of *pre-defined bits* at the end of an allocated memo
 
 On every operation, we would check that the sentinel value is still the same as it was pre-defined. If the sentinel value has changed, then we know for certain that the user has corrupted memory.
 
-In your `mini_memcheck`, you will use the [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) `0xCAFEBABE` for sentinel value. [(Why x0CAFEBABE?)](http://www.artima.com/insidejvm/whyCAFEBABE.html)
```

Comment:
Wondering if we need to delete the link too-- could be interesting to keep as a 'historical magic number'/informational link, although I get why we may just want to destroy all references.

Up to your discretion.

Reply:
I think I'll leave it removed. IMO, the wikipedia entry has enough history/info about any set of magic numbers.